README.md

```diff
@@ -51,11 +51,11 @@ This table only lists the options used in this role, see [xen-vman](https://gith
-| `zfs_attributes` |           `{}`           | A dict containing any ZFS attributes that shall be set for this filesystem. It is recommended to define a `quota`, though this may also be done in the default ZFS attributes on the actual [zfs-storage](https://github.com/stuvusIT/zfs-storage/) server.                                                                                         |
-| `nfs_options`    | `[rw=@{{interface.ip}}]` | A list of NFS options that are set for this NFS export. The default is an rw access for every IP defined in (`vm.interfaces`)[https://github.com/stuvusIT/xen_vman#vm-interfaces]. The options have to conform to the ZFS `sharenfs` attribute format. The options defined in `vm_nfs_options` will be set in addition to this value in every case. |
+| Name             |   Default / Mandatory    | Description                                                                                                                                                                                                                                                                                                                                    |
+|:-----------------|:------------------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `name`           |    :heavy_check_mark:    | `name`-suffix of the filesystem. The final name consists of the vm name and the filesystem name, delimited by a dash.                                                                                                                                                                                                                          |
+| `zfs_attributes` |           `{}`           | A dict containing any ZFS attributes that shall be set for this filesystem. It is recommended to define a `quota`, though this may also be done in the default ZFS attributes on the actual [zfs-storage](https://github.com/stuvusIT/zfs-storage/) server.                                                                                    |
+| `nfs_options`    | `[rw=@{{ansible_host}}]` | A list of NFS options that are set for this NFS export. The default is one read-write NFS export for the IP defined in (`ansible_host`) in the hostvars of the respective VM. The options have to conform to the ZFS `sharenfs` attribute format. The options defined in `vm_nfs_options` will be set in addition to this value in every case. |
```

Comment:
why the fqdn and not ansible_host_short?

Reply:
`ansible_host` is the IP of the respective host.

Reply:
@Mr-Pi From the inventory:

```
hostConfig['ansible_host'] = interface['ip'].split('/')[0]
```